checks.d/marathon.py

```diff
@@ -56,6 +56,10 @@ def check(self, instance):
                     if attr in app:
                         self.gauge('marathon.' + attr, app[attr], tags=tags)
 
+        response = self.get_json(urljoin(url, "v2/deployments"), timeout, auth)
```

Comment:
I'm cool with tossing this behind a config flag. I'll push up a commit for that momentarily.